app/controllers/mail_controller.rb

```diff
@@ -0,0 +1,17 @@
+class MailController < ApplicationController
+  before_action :admin_required
+
+  def write_email
+    @simplemail = SimpleMail.new 
```

Comment:
Powinien zostać również ustawiony nadawca oraz adresat:
```ruby
@simplemail.sender = current_user.email
@simplemail.addressee = Person.find(params[:id])
```
Żeby to jednak zadziałało trzeba wprowadzić zmiany w pliku `routes.rb`